btb/tuning/hyperparams/numerical.py

```diff
@@ -0,0 +1,235 @@
+# -*- coding: utf-8 -*-
+
+"""Package where the numerical hyperparameters are defined."""
+
+import sys
+
+import numpy as np
+
+from btb.tuning.hyperparams.base import BaseHyperParam
+
+
+class NumericalHyperParam(BaseHyperParam):
+    """Numerical Hyperparameter Class.
+
+    The numerical hyperparameter class defines an abstraction to hyperparameters which ranges are
+    defined by a numerical value and can take any number between that range.
+
+    Attributes:
+        K (int):
+            Number of dimensions that this hyperparameter uses to be represented in the search
+            space.
+
+    Args:
```

Comment:
Let's document the Args in the subclasses where the `__init__` method is defined instead of here.